packages/ember-glimmer/tests/integration/content-test.js

```diff
@@ -73,6 +73,8 @@ class DynamicContentTest extends RenderingTest {
   ['@test it can render a dynamic path']() {
     this.renderPath('message', { message: 'hello' });
 
+    expectNoDeprecation();
```

Comment:
To cover [this test](https://github.com/emberjs/ember.js/blob/caa99153e6abb26b186f3c2beb19b20694e6858c/packages/ember-htmlbars/tests/integration/escape_integration_test.js#L24). Not sure if this is necessary anymore. Looks like its the last piece related to deprecated global lookup (86fa0ab)?